framework/src/controller/migrations/migration.js

```diff
@@ -218,7 +224,19 @@ class Migration extends BaseEntity {
 	 *
 	 * @returns {Promise<boolean>} Promise object that resolves with a boolean.
 	 */
-	async hasMigrations() {
+	async hasInternalMigrationsTable() {
+		const hasMigrations = await this.adapter.execute(
+			"SELECT table_name hasMigrations FROM information_schema.tables WHERE table_name = 'internal_migrations';"
```

Comment:
Its bad practice to use mixed case name without quotes. 

```suggestion
			"SELECT table_name \"hasMigrations\" FROM information_schema.tables WHERE table_name = 'internal_migrations';"
```